src/app/[locale]/(main)/quizbank/[id]/components/recommend-quizbank.tsx

```diff
@@ -0,0 +1,164 @@
+              Teacher: Eva Doe
+            </div>
+            <div className="text-xs leading-4 text-zinc-500">
+              eva@example.com
+            </div>
+          </div>
+        </div>
+        <div className="flex justify-between gap-2">
+          <TooltipProvider>
+            <Tooltip>
+              <TooltipTrigger asChild>
+                <Button variant="light" color={null}>
+                  <CopyIcon />
+                </Button>
+              </TooltipTrigger>
+              <TooltipContent>
+                <p>{i18n("author.copy_button")}</p>
+              </TooltipContent>
+            </Tooltip>
+          </TooltipProvider>
+          <TooltipProvider>
+            <Tooltip>
+              <TooltipTrigger asChild>
+                <Button variant="light" color={null}>
+                  <InfoCircledIcon />
+                </Button>
+              </TooltipTrigger>
+              <TooltipContent>
+                <p>{i18n("author.report_button")}</p>
+              </TooltipContent>
+            </Tooltip>
+          </TooltipProvider>
+        </div>
+      </div>
+
+      {/* footer, link to another */}
+      <div className="mt-80 flex w-full flex-col items-start justify-center self-stretch bg-white px-11 py-7 text-base max-md:mt-10 max-md:max-w-full max-md:px-5">
+        <div className="flex w-[923px] max-w-full justify-between gap-5 max-md:flex-wrap">
+          <div className="flex flex-col">
+            <div className="flex justify-between gap-1.5 whitespace-nowrap text-center tracking-widest text-black">
+              <img
+                loading="lazy"
+                src="https://cdn.builder.io/api/v1/image/assets/TEMP/86b70a3d23b6e625c9aab5ac0ce591e471f393f213e877b530e86aaca7d8b45d?"
+                className="aspect-square w-6"
+              />
+              <div className="flex-auto">Quizlearn</div>
+            </div>
+            <div className="leading-6 text-zinc-900">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas
+              mi risus, lobortis sit amet urna ac, aliquam interdum nibh.
+              Phasellus vestibulum tincidunt tellus,{" "}
+            </div>
+          </div>
+          <div className="flex flex-col self-start py-1.5 font-semibold leading-[150%] text-zinc-900">
+            <div className="flex justify-between gap-5 px-0.5">
+              <div>Link 1</div>
+              <div>Link 1</div>
+              <div>Link 1</div>
+            </div>
+            <div className="mt-7 flex justify-between gap-5 px-0.5">
+              <div>Link 1</div>
+              <div>Link 1</div>
+              <div>Link 1</div>
+            </div>
+            <div className="mt-7 flex justify-between gap-5 px-0.5">
+              <div>Link 1</div>
+              <div>Link 1</div>
+              <div>Link 1</div>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
Fotter use `<footer>` tag and should be in another component and place inside `layout.tsx`